scalalib/src/mill/scalalib/PublishModule.scala

```diff
@@ -73,32 +73,35 @@ trait PublishModule extends JavaModule { outer =>
   }
 
   def publish(sonatypeCreds: String,
-              gpgPassphrase: String,
+              gpgPassphrase: String = "",
```

Comment:
Is it possible to make this argument optional? Same applies to `publishAll` command

Reply:
I don't think it's possible to write a ScoptReader for "absence of values" tbh. However your suggestion to default to null is better than empty string. Addressed : ec362d8